src/Mesh/Mesh.cc

```diff
@@ -845,7 +845,7 @@ generateDomainInfo() {
 
   // Determine which process owns each shared node.  We use the convention that the process
   // with the lowest rank wins.
-  unordered_map<unsigned, unsigned> nodeOwner;
+  boost::unordered_map<unsigned, unsigned> nodeOwner;
```

Comment:
Is there a big reason for switching to boost::unordered_map here?

Reply:
I believe we have always been using the boost implementation here. `using namespace boost` & `using ::boost::unordered_map` is included at the top of this file. I think I added it for clarity on which `unordered_map` we are calling at the moment.